target_postgres/stream_tracker.py

```diff
@@ -0,0 +1,74 @@
+from collections import deque
+import json
+import singer.statediff as statediff
+import sys
+
+from target_postgres.exceptions import TargetError
+
+
+class StreamTracker:
+    """
+    Object to track STATE messages coming in from the tap and which streams need to be flushed before they can be safely emitted.
```

Comment:
nit: Should this be updated to reflect the new class? It does more than tracking state messages.